v0/destinations/pinterest_tag/transform.js

```diff
@@ -180,7 +118,18 @@ const process = event => {
     case EventType.PAGE:
     case EventType.SCREEN:
     case EventType.TRACK:
-      mandatoryPayload = commonFieldResponseBuilder(message, destination);
+      deducedEventNameArray = deduceEventName(message, destination.Config);
```

Comment:
```suggestion
      deducedEventNameArray.push(
        ...deduceEventName(message, destination.Config)
      );
```

Reply:
@ItsSudip what's the need for exploding array & again pushing it to new array ?

Reply:
I just wanted to make the array constant. That is why apart from creating a new array I asked to push in the existing array.